src/lang/modifyAst/faces.ts

```diff
@@ -181,64 +182,107 @@ export function addHole({
 
   const { solidsExpr, facesExpr, pathIfPipe } = result
 
+  // Extra args for createCallExpressionStdLibKw as we're calling functions from a module
+  const nonCodeMeta = undefined
+  const modulePath = [createIdentifier('hole')]
+
   // Prep the big label args
   let holeBodyNode: Node<CallExpressionKw> | undefined
   if (holeBody === 'blind' && blindDepth && blindDiameter) {
-    holeBodyNode = createCallExpressionStdLibKw('hole::blind', null, [
```

Comment:
Explicitely doing `hole::blind` was wrong, as in `createLocalName` it would only set the `name` prop to `hole::blind`, instead of `name` to `blind` and `path` (which I didn't know about) to `[hole]`!